splunk/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM docker.io/splunk/splunk@sha256:0bcd9c9f836cfb72b80b5ff8deec692b561473e68b9e41ea5c1f7d92abc35c61 AS builder
+
+ENV SPLUNK_START_ARGS=--accept-license
+
+USER root
+
+COPY --chmod=777 tests/test_logs /opt/splunk/test_logs
+COPY --chmod=777 tests/indexes.conf /opt/splunk/etc/apps/search/local/indexes.conf
+COPY --chmod=777 scripts/log_indexing.sh /opt/splunk/log_indexing.sh
+COPY --chmod=777 scripts/pre-run.sh /opt/splunk/pre-run.sh
+
+RUN microdnf install -y jq \
+    && microdnf clean all \
+    && umask 000 \
+    && touch "/opt/splunk/output.log"
+
+USER ansible
+RUN bash -o errexit /opt/splunk/pre-run.sh
+
+FROM docker.io/splunk/splunk@sha256:0bcd9c9f836cfb72b80b5ff8deec692b561473e68b9e41ea5c1f7d92abc35c61
+
+ENV SPLUNK_START_ARGS=--accept-license
+
+EXPOSE 8080
+EXPOSE 8089
+
+USER root
+
+RUN mkdir -p /opt/splunk/var/lib/splunk
+COPY --chmod=777 --from=builder /opt/splunk/etc/apps/search/local/indexes.conf /opt/splunk/etc/apps/search/local/indexes.conf
```

Comment:
777? seriously? it should at least be 666 or 665 or probably 660 or even 600

Reply:
Anything with less permissions than 777 leads to errors during runtime (Ansible playbook fails).